environment.yml

```diff
@@ -0,0 +1,17 @@
+name: datapipes
+channels:
+  - pytorch
+  - conda-forge
+  - defaults
+dependencies:
+  - pip
+  - pytorch
+  - rioxarray
+  - torchdata
+  - torchvision
+  - xarray
+  - fsspec
+  - zarr
+pip:
+  - einops
+  - pathy
```

Comment:
are we sure we want enviornment? Is requirements.txt not enough? Or perhaps there are some other funny things